src/main/java/tconstruct/modifiers/tools/ModAntiSpider.java

```diff
@@ -1,19 +1,19 @@
 package tconstruct.modifiers.tools;
 
 import net.minecraft.item.ItemStack;
+import net.minecraft.util.StatCollector;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.StatCollector;
 
 public class ModAntiSpider extends ItemModTypeFilter
 {
     String tooltipName;
     int max = 4;
-    String guiType;
 
-    public ModAntiSpider(String type, int effect, ItemStack[] items, int[] values)
+    public ModAntiSpider(int effect, ItemStack[] items, int[] values)
```

Comment:
Don't change the signature of modifiers or it might cause incompatibilites with other mods using it.